components/navigation/JobPostItem.js

```diff
@@ -1,46 +1,92 @@
+import { join } from "lodash";
+
 export default function JobPostItem({ job }) {
-  return (    
+  return (
     <li>
       <a href={job.slug} className="block hover:bg-gray-50">
         <div className="px-4 py-4 sm:px-6">
           <div className="flex items-center justify-between">
             <p className="text-sm font-medium text-indigo-600 truncate">
-              { job.title }
+              {job.title}
             </p>
+            <a href={job.company.url}>
```

Comment:
I was just thinking about the design and had this idea:

![Captura de pantalla 2021-05-08 a las 14 09 58](https://user-images.githubusercontent.com/242119/117538745-e77e3600-b007-11eb-9deb-d3d667ff24e0.png)

If we do it this way, we don't have to link to the company website. This information can be inside the job posting.

It would be just a matter of adding `at {job.company.name}` and the logo before the title. What do you think?

Reply:
![image](https://user-images.githubusercontent.com/56452820/117546414-ef5dca80-b047-11eb-9278-50d70e24adc4.png)
I have changed the design .